src/main/java/com/elytradev/smores/Smores.java

```diff
@@ -1,100 +1,170 @@
+import com.elytradev.smores.block.BlockBase;
+import com.elytradev.smores.block.BlockGem;
+import com.elytradev.smores.block.BlockGemOre;
+import com.elytradev.smores.block.BlockMetal;
+import com.elytradev.smores.block.BlockMetalOre;
+import com.elytradev.smores.block.BlockNetherOre;
+import com.elytradev.smores.generic.IOreDict;
 import com.elytradev.smores.generic.SmoresCreativeTab;
+import com.elytradev.smores.init.SmoresBlocks;
+import com.elytradev.smores.item.ItemBase;
+import com.elytradev.smores.item.ItemBlockSubtyped;
+import com.elytradev.smores.item.ItemDust;
+import com.elytradev.smores.item.ItemGear;
+import com.elytradev.smores.item.ItemGem;
+import com.elytradev.smores.item.ItemIngot;
+import com.elytradev.smores.item.ItemNugget;
+import com.elytradev.smores.item.ItemPlate;
+import com.elytradev.smores.materials.EnumAlloy;
+import com.elytradev.smores.materials.EnumGem;
+import com.elytradev.smores.materials.EnumMetal;
+import com.elytradev.smores.materials.EnumNether;
 import com.elytradev.smores.proxy.CommonProxy;
-import com.elytradev.smores.registries.*;
+import com.elytradev.smores.registries.ConfigurationRegistry;
+import com.elytradev.smores.registries.FluidRegistry;
+import com.elytradev.smores.registries.RecipeRegistry;
 import com.elytradev.smores.world.WorldGen;
-import com.google.common.collect.ImmutableList;
+import net.minecraft.block.Block;
+import net.minecraft.creativetab.CreativeTabs;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemBlock;
+import net.minecraft.util.ResourceLocation;
 import net.minecraftforge.common.config.Configuration;
+import net.minecraftforge.event.RegistryEvent;
 import net.minecraftforge.fml.common.Mod;
 import net.minecraftforge.fml.common.SidedProxy;
 import net.minecraftforge.fml.common.event.FMLInitializationEvent;
-import net.minecraftforge.fml.common.event.FMLInterModComms;
 import net.minecraftforge.fml.common.event.FMLPostInitializationEvent;
 import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
 import net.minecraftforge.fml.common.registry.GameRegistry;
+import net.minecraftforge.fml.common.registry.IForgeRegistry;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
+@Mod(
+        modid = SmoresConstants.MOD_ID,
+        name = SmoresConstants.NAME,
+        version = SmoresConstants.VERSION,
+        guiFactory = "com.elytradev.smores.gui.SmoresGuiFactory")
+@Mod.EventBusSubscriber
+public final class Smores {
 
-@Mod(modid = Smores.SMORES_MOD_ID, name = Smores.SMORES_NAME, version = Smores.SMORES_VERSION, guiFactory = "com.elytradev.smores.gui.SmoresGuiFactory")
-public class Smores {
-    public static final String SMORES_MOD_ID = "smores";
-    public static final String SMORES_NAME = "Smores";
-    public static final String SMORES_VERSION = "1.11.2-1.0.0.0";
+    public static final Logger log = LogManager.getLogger(SmoresConstants.NAME);
```

Comment:
Using lowercase for `Logger`s is a convention I have long been accustomed to. Forge also follow this convention - https://github.com/MinecraftForge/MinecraftForge/blob/1.12.x/src/main/java/net/minecraftforge/fml/common/FMLLog.java#L40.